dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -46,6 +46,11 @@ programs=
 isc-dhcp-relay-{{ vlan_name }}
 {%- endif %}
 {% endfor %}
+{% for vlan_name in VLAN %}
+{% if VLAN[vlan_name]['dhcp_servers'] %}
+,dhcpmon-{{ vlan_name }}
```

Comment:
dhcpmon processes should not be added to the isc-dhcp-relay group. You should create a new "dhcpmon" group and add them to it instead. This was they can be managed independently of one another.

Reply:
done